src/main/java/io/apicurio/datamodels/core/models/common/INamed.java

```diff
@@ -16,9 +16,23 @@
 
 package io.apicurio.datamodels.core.models.common;
 
+import io.apicurio.datamodels.core.models.IVisitable;
+
 /**
+ * Any node model that has a name.
  * @author eric.wittmann@gmail.com
  */
-public interface ISecuritySchemaDefinition extends IDefinition {
+public interface INamed extends IVisitable {
```

Comment:
I've seen `IDefinition` used in visitors, could it extend both `INamed` and `IVisitable` directly?

Reply:
This is a good point - it's a little weird, and was only done because it eliminated the need for some casting in a few places.  But I'll try removing it and seeing how annoying the casting really is. :)